build-system/tasks/bundle-size/APPROVERS.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
This is just an example so we can get started. Later on we'll let teams opt-in to this

Reply:
Who will be the owners of this?

Reply:
For now let's leave the existing ownership model (wg-infra) - we can change it later

Reply:
I'm fine with wg-infra remaining the owner too